pkg/cmd/server/origin/ensure.go

```diff
@@ -144,7 +144,8 @@ func (c *MasterConfig) ensureNamespaceServiceAccountRoleBindings(namespace *kapi
 		namespace.Annotations = map[string]string{}
 	}
 	namespace.Annotations[ServiceAccountRolesInitializedAnnotation] = "true"
-	if _, err := c.KubeClient().Namespaces().Update(namespace); err != nil {
+	// Log any error other than a conflict (the update will be retried and recorded again on next startup in that case)
+	if _, err := c.KubeClient().Namespaces().Update(namespace); err != nil && !kapierror.IsConflict(err) {
```

Comment:
I'd be happier with using a patch.  Do we have an issue for easily creating patches on `runtime.Objects`?